app/_tutorials/add-rate-limiting-for-a-consumer-with-kong-gateway.md

```diff
@@ -42,10 +42,6 @@ prereqs:
         - example-service
     routes:
         - example-route
-    consumers:
-        - key-auth-consumer
-    plugins:
-        - key-auth
```

Comment:
@lmilan can you add `key-auth` to the list of plugins in the metadata?

Reply:
done!